spec/services/create_batch_spec.rb

```diff
@@ -39,7 +39,6 @@ def create_appointment_summary
       end
 
       include_examples 'ignore unsupported appointment summaries', format_preference: 'braille'
-      include_examples 'ignore unsupported appointment summaries', format_preference: 'large_text'
```

Comment:
We shouldn't be including large text in the batch yet since the upstream provider doesn't yet have that capability?